libwild/src/resolution.rs

```diff
@@ -661,7 +661,7 @@ fn load_prelude<'definitions>(
     // symbol record to the definition.
     for (def_info, definition_out) in prelude.symbol_definitions.iter().zip(definitions_out) {
         match def_info.placement {
-            SymbolPlacement::ForceUndefined => {
+            SymbolPlacement::ForceUndefined | SymbolPlacement::DefsymSymbol(_) => {
                 load_symbol_named(resources, definition_out, def_info.name, work_sender);
```

Comment:
Is `def_info.name` the right name to be loading here? I thought that was the alias and that you'd actually want to load the target symbol?